about.html

```diff
@@ -0,0 +1,134 @@
+      rel="stylesheet"
+    />
+    <link rel="icon" href="./Images/logo-no-background.svg" type="image/svg" />
+  </head>
+<body>
+    <section class="nav-bar">
+        <div id="ham-menu">
+          <img
+            onclick="clickX()"
+            id="x"
+            src="https://img.icons8.com/sf-regular/48/FFFFFF/multiply.png"
+          />
+          <ul class="ham">
+            <li>
+              <a href="index.html" class="ham-items" onclick="clickX()"
+                >Home</a
+              >
+            </li>
+            <li>
+              <a href="about.html" class="ham-items" onclick="clickX()">About</a>
+            </li>
+            <li>
+              <a href="#contact" class="ham-items" onclick="clickX()">Program</a>
+            </li>
+            <li>
+              <a href="#contact" class="ham-items" onclick="clickX()">Join</a>
+            </li>
+            <li>
+              <a href="#contact" class="ham-items" onclick="clickX()">Sponsor</a>
+            </li>
+          </ul>
+        </div>
+        <header class="head flex-row">
+          <nav>
+            <img
+              onclick="clickMenu()"
+              id="menu"
+              src="./Images/Icon - Menu.png"
+              alt="Menu Logo"
+              class="menu-logo"
+            />
+            <a href="">
+              <img class="logo" src="Images/logo-black.png" alt="logo" />
+            </a>
+            <ul class="menu_items flex-row">
+              <li class="nav_items"><a href="index.html">Home</a></li>
+              <li class="nav_items"><a href="about.html">About</a></li>
+              <li class="nav_items"><a href="">Program</a></li>
+              <li class="nav_items"><a href="">Join</a></li>
+              <li class="nav_items"><a href="">Sponsor</a></li>
+            </ul>
+          </nav>
+        </header>
+        <div class="main-headline">
+          <h3 class="welcome">"JOIN US"</h3>
+          <h3 class="welcome-2">Take A Step </h3>
+          <h1 class="Meco">MECO</h1>
+          <h3 class="bottom">
+            On July 13 We are Coming !!!!
+          </h3>
+        </div>
+    </section>
+    <section class="contact">
+      <div class="mail">
+        <p>Please contact us per Email for nay further questions about CC Global Summit 2023!</p>
+        <a href="#">mecoworld.meco.cd</a>
+      </div>
+      <div>
+        <div class="logo_des">
+          <h2>MECO Global Summit 2023 Logo</h2>
+          <div class="red-bar"></div>
+          <p>The logo of MECO Global Summit 2015 was decided through the logo competition from 8, June to 7, July.
+            After online public voting and Selection Committee’s judging process, the logo from Shihab Uddin Khan won the prize.
+            After consultation with Shihab Uddin Khan, we optimized the prize-winning-logo for this year’s summit as follows:</p>
+        </div>
+        <a href="">
+          <img class="logo-1" src="Images/logo-color.svg" alt="logo" />
```

Comment:
- [ ] your logo seems to be broken. Kindly adjust it
![Screenshot (985)](https://user-images.githubusercontent.com/65298048/233437757-795bd543-4eea-4eb5-8848-00e700088e94.png)

Reply:
On my side, Logo is already showing.